astropy/utils/tests/test_data_info.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# TEST_UNICODE_LITERALS
+
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+from ...extern import six
+from ..data_info import dtype_info_name
+from ...tests.helper import pytest
+
+STRING_TYPE_NAMES = {(False, 'S'): 'str',  # not PY3
+                     (False, 'U'): 'unicode',
+                     (True, 'S'): 'bytes', # PY3
+                     (True, 'U'): 'str'}
+
+DTYPE_TESTS = ((np.array(b'abcd').dtype, STRING_TYPE_NAMES[(six.PY3, 'S')] + '4'),
```

Comment:
Really very minor, so feel free to ignore, but personally I like the tests to be very explicit and easy to read, so I would do:
```
DTYPE_TESTS = ((np.array(b'abcd').dtype, 'bytes4' if six.PY3 else 'str4'),
               (np.array(u'abcd').dtype, 'str4' if six.PY3 else 'unicode4'),
               ('S4', 'bytes4' if six.PY3 else 'str4'),
               ('U4', 'str4' if six.PY3 else 'unicode4'),
...
```
Obviously, with that you can also delete the `STRING_TYPE_NAMES` dict (which, if you do not change this, you might as well import).